app/src/app/[lng]/[inventory]/InventoryResultTab/index.tsx

```diff
@@ -160,9 +165,19 @@ function SectorTabs({
                       )}
                     </Select>
                   </Box>
+                  )
                 </HStack>
-                {isResultsLoading && <CircularProgress />}
-                {!isResultsLoading &&
+                {isResultsLoading && <CircularProgress isIndeterminate />}
+                {isEmptyInventory && (
+                  <EmptyStateCardContent
+                    t={t}
+                    inventoryId={inventory.inventoryId}
+                    width={"1042px"}
+                    height={"592px"}
```

Comment:
Ideally we don't hard code sizes in pixels as that breaks the page for anyone using larger or smaller font settings and isn't great for responsiveness most of the time.

Fine for now but we have to start thinking about our pages less like exact copies of the designs and more like responsive documents...